components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -710,7 +710,10 @@ void RewardsServiceImpl::OnUnload(SessionID tab_id) {
   if (!Connected())
     return;
 
-  bat_ledger_->OnUnload(tab_id.id(), GetCurrentTimestamp());
+  bat_ledger_->OnUnload(
+      tab_id.id(),
+      GetCurrentTimestamp(),
+      base::DoNothing());
```

Comment:
Some of `On`action methods call into media publishers and consequently required a callback. Recommend some other action other than `base::DoNothing` if appropriate.